modules/serverless-about-collecting-data.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-support.adoc
+
+[id="serverless-about-collecting-data_{context}"]
+= About collecting {ServerlessProductName} data
+
+You can use the `oc adm must-gather` CLI command to collect information about your cluster, including features and objects associated with {ServerlessProductName}.
+
+To collect {ServerlessProductName} data with `must-gather`, you must specify the {ServerlessProductName} image:
+
+----
+$ oc adm must-gather --image=quay.io/openshift-knative/must-gather
```

Comment:
AFAICT, the final location of the productized image will be: `registry.redhat.io/openshift-serverless-1/svls-must-gather-rhel8` so this command should be changed to `oc adm must-gather --image=registry.redhat.io/openshift-serverless-1/svls-must-gather-rhel8`